democracylab/settings.py

```diff
@@ -400,9 +399,6 @@ def read_connection_config(config):
 
 # If you're using prerender.io (the default backend):
 SEO_JS_PRERENDER_TOKEN = os.environ.get('SEO_JS_PRERENDER_TOKEN', '')
-SEO_JS_BACKEND = "common.helpers.caching.DebugPrerenderIO"
```

Comment:
Anything prefixed with SEO_JS_ is something we should be able to get rid of.

Reply:
I got a `ValueError: Missing SEO_JS_PRERENDER_TOKEN in settings.` error when I tried to boot the containers after removing those values.

Reply:
Did you try it after removing the library references from requirements.txt and MIDDLEWARE?